logging/example_logging.py

```
@@ -1,9 +1,40 @@
+import os
+import time
 import logging
 
+
+def reset_log(log_file):
+    try:
+        os.remove(log_file)
+    except OSError:
+        pass
```

Is it good approach to silently pass error?